src/java/org/apache/cassandra/metrics/KeyspaceMetrics.java

```diff
@@ -233,8 +233,8 @@ public KeyspaceMetrics(final Keyspace ks)
         confirmedRepairedInconsistencies = createKeyspaceMeter("RepairedDataInconsistenciesConfirmed");
         unconfirmedRepairedInconsistencies = createKeyspaceMeter("RepairedDataInconsistenciesUnconfirmed");
 
-        repairedDataTrackingOverreadRows = createKeyspaceHistogram("RepairedOverreadRows", false);
```

Comment:
for me, this doesn't need to be aliased, see https://issues.apache.org/jira/browse/CASSANDRA-15909?focusedCommentId=17150195&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17150195